colortv/docs/androidTVSDK.md

```diff
@@ -134,220 +129,295 @@ ColorTvContentRecommendationListener listener = new ColorTvContentRecommendation
-    
-##Displaying Ads
+###Showing Content Recommendations
 
-!!! note "WARNING"
-    Ads are provided only for AndroidTv devices. 
+You can show Content Recommendations in one of two visual forms: **Recommendation Center** and **UpNext**.
 
-Ads may be shown wherever you place them inside your app, but they **must** include a Placement parameter to indicate the specific location. Placements are used to optimize user experience and analytics. 
+####Recommendation Center
 
-You can use the same Placements as are pointed in [content recommendation section](#placements).
- 
-!!! note ""
-    You can choose what ad units you want to show for a specific placement in the dashboard, [click to learn more about Ad Units](index.md#ad-units)
-   
-!!! note ""
-    The `AdPlacement` class has been deprecated and replaced by `Placements`. The old version will be removed in future versions of the SDK.
-    
-To get callbacks about the ad status, you need to create a `ColorTvAdListener` object by overriding it's methods:
+Recommendation Center is a unit that lets you display recommendations in an Activity with scrollable grid layout form.
+
+In order to show Recommendation Center, you have to call following method: 
 
 ```java
-ColorTvAdListener listener = new ColorTvAdListener() {
+colorTvRecommendationsController.showRecommendationCenter(Placements.VIDEO_END);
+```
 
-    @Override
-    public void onAdLoaded(String placement) {
-        ColorTvSdk.showAd(placement);
-    }
+Invoking this method will show Recommendation Center for the placement you pass. You need to call the `load` method for a given placement before invoking `show` in order to load recommendations related data. Also make sure you got the `onLoaded` callback first, otherwise the Recommendation Center won't be displayed.
 
-    @Override
-    public void onAdError(String placement, ColorTvError colorTvError) {
-    }
+####UpNext
 
-    @Override
-    public void onAdClosed(String placement, boolean watched) {
-    }
+UpNext is a unit which diplays only the best recommendation in form of a small view designed to be placed above a video, which up next holds recommendation for. In order to keep the video playing we are delivering UpNext as a Fragment which you need to add to your layout. To make it work properly, you need to add a container to which you'll inject the UpNext fragment. This container have to be inside a RelativeLayout or a FrameLayout with the possibility to use the whole screen size in order to let it place itself in a proper place and size for various devices. The container should also have width set to `match_parent` and height set to `wrap_content`. For a sample of the correct layout, please refer to our [sample application's layout](https://github.com/color-tv/android-SampleApp/blob/master/SampleApp/app/src/main/res/layout/activity_exo.xml). In close future we are going to provide more possibilities to customize the UpNext layout and it's position.
```

Comment:
'designed to be placed above a video' I think 'designed to be placed over a video' could be better.